server/routes/api/auth.js

```diff
@@ -0,0 +1,128 @@
+var router = require('express').Router(),
+    userModel = require('../../db/model/User.js'),
+    authMiddleware = require('../../middleware/auth.js');
+
+// Disable all non-post methods for /v1/auth
+router.use(function(req, res, next) {
+    if (req.method !== 'POST') {
+        res.status(405).send({
+            status: false,
+            message: "Method not allowed"
+        });
+    }
+    return next();
+});
+
+// Handles /v1/auth/login
+router.post('/login', function(req, res) {
+    // Check if the login request contains the email and password
+    if (req.body.email && req.body.password) {
```

Comment:
What I want to do is add a garbage collector util in that will periodically run to verify db tokens (and therefore remove old ones). We can invalidate tokens if they go over a maximum amount too if wanted.